doc/source/beginner/overview.rst

```diff
@@ -69,14 +69,70 @@ qibuild is composed of two parts:
   taking dependencies into account and generate re-distributable binary
   packages.
 
+What makes qiBuild different ?
+++++++++++++++++++++++++++++++
 
-Is qibuild the only one build framework?
-++++++++++++++++++++++++++++++++++++++++
+The good parts
+~~~~~~~~~~~~~~
 
+* Full support for Visual Studio
 
-Of course not!
+* Full support for cross-compilation (hosts: linux, mac: targets : x86, arm, android, ...)
 
-You can have a loot at
+* Comes with a tool to use pre-compiled dependencies
+
+* No environment variables required, and keep your environment clean
+
+* Written in Python like many others but:
+
+  * Python2/Python3 compatible
+
+  * >80% test coverage
+
+* Can find dependencies **from the sources**. For instance, in a worktree with
+  two different CMake projects , ``world`` and ``hello``, when compiling ``hello``,
+  we will find ``world`` headers directly from ``world`` sources.
+
+* Automatic install rules (you have to *explicitely* exclude targets from installation)
+
+* ``qibuild package --standalone`` generates an archive that is:
+
+  * relocatable
+  * and work across linux distributions
+
+* Full Python support : you can write Python code that use extensions written in C++
+  with ease, and run the C++ and the Python tests with the same tool
+
+* Your build dir contains a nice ``sdk`` folder with files where you expect them
+  (``.dlls`` and ``.exe`` in ``sdk/bin``, ``.so`` in ``sdk/lib/`` and data in
+  ``sdk/share``)
+
+* You can deploy your code to a remote host via ``ssh`` and ``rsync``
+
+* You can also deploy or install your tests, and then only run them with
+  ``qitest``
+
+The bad parts
+~~~~~~~~~~~~~
+
```

Comment:
maybe the fact that qibuild scatters its products in several sdk directories then requires magic (even runtime magic for findData) should be mentioned.